packages/scene-composer/src/components/panels/SceneHierarchyPanel/components/SceneHierarchyTree/SceneHierarchyTreeItem.tsx

```diff
@@ -94,7 +94,14 @@ const SceneHierarchyTreeItem: FC<SceneHierarchyTreeItemProps> = ({
   return (
     <EnhancedTreeItem
       key={key}
-      labelNode={<SceneNodeLabel objectRef={key} labelText={labelText} componentTypes={componentTypes} />}
+      labelNode={
+        <SceneNodeLabel
+          dataTestid={node?.components[0].ref}
```

Comment:
Why the first component ref and not the node ref itself?

Reply:
The `component` ref is the Three.js object ref. The `node` ref has no association to the Three.js object. Using `component` in order to validate that it is the correct 3D object being represented in the Inspector Panel.